airbyte-integrations/connectors/source-amplitude/source_amplitude/api.py

```diff
@@ -21,10 +21,17 @@
 
 
 class AmplitudeStream(HttpStream, ABC):
-
-    url_base = "https://amplitude.com/api/"
     api_version = 2
 
+    def __init__(self, data_region: str, **kwargs):
+        self.data_region = data_region
+        super().__init__(**kwargs)
+
+    @property
+    def url_base(self) -> str:
+        subdomain = "analytics.eu." if self.data_region == "EU Residency Server" else ""
```

Comment:
Are there more than EU + US?

Reply:
No according to the documentation https://www.docs.developers.amplitude.com/analytics/apis/http-v2-api/